setup.py

```diff
@@ -30,10 +30,14 @@ def read_version(package):
 ]
 
 swagger_ui_require = 'swagger-ui-bundle>=0.0.2,<0.1'
-flask_require = 'flask>=1.0.4,<3'
+flask_require = [
+    'flask>=1.0.4,<3',
+    'itsdangerous>=0.24'
```

Comment:
minor nit: but adding a trailing comma to the last element of the list makes future diffs a bit nicer